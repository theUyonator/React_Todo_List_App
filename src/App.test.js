import React from "react";
import { render } from "@testing-library/react";
import App from "./App";

describe("<App/> rendering", () => {

it("renders correctly without crashing", () => {
    render(<App />)
});

it("matches snapshot", () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
});

});