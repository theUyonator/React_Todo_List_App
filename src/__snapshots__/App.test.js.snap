// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> rendering matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="TodoList"
  >
    <form>
      <label
        for="todo"
      >
        Todo Item:
      </label>
      <input
        id="todo"
        name="todo"
        value=""
      />
      <button>
        Add a new todo!
      </button>
    </form>
    <ul />
  </div>
</DocumentFragment>
`;
